client/src/App.js

```diff
@@ -1,5 +1,5 @@
-import React, { Component } from 'react';
-import { Route, Switch, Redirect, BrowserRouter } from 'react-router-dom';
+import React, { Component } from "react";
```

Comment:
our code style is singlequote.

Reply:
Weird! I have no idea how that happened since I shouldn't have touched that code 😅 Should be all fixed now!